### PR TITLE
reitroduce the deleted cask

### PR DESCRIPTION
**Pull request by nufer**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/kleiner-brauhelfer.app.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'kleiner-brauhelfer.app' do
+  version '1.4.4.5'
```
Comment (miccal) [inline Casks/kleiner-brauhelfer.app.rb:2]: Should this not be `1.4.4.6`?

File: Casks/kleiner-brauhelfer.app.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  # github.com/realholgi/kleiner-brauhelfer was verified as official when first introduced to the cask
+  url "https://github.com/realholgi/kleiner-brauhelfer/releases/download/v#{version}/kb_macos_v#{version.dots_to_underscores}.zip"
+  appcast 'https://github.com/realholgi/kleiner-brauhelfer/releases.atom'
+  name 'Kleiner Brauhelfer'
+  homepage 'http://www.joerum.de/kleiner-brauhelfer/doku.php'
+
+  app 'kleiner-brauhelfer.app'
+
+  zap trash: 'brauhelfer.app'
```
Comment (miccal) [inline Casks/kleiner-brauhelfer.app.rb:13]: Please remove this -- it is not required.

File: Casks/kleiner-brauhelfer.app.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'kleiner-brauhelfer.app' do
```
Comment (miccal) [inline Casks/kleiner-brauhelfer.app.rb:1]: Just `kleiner-brauhelfer`, not `kleiner-brauhelfer.app`.

**Comment by vitorgalvao** [comment]
[This was removed for a reason](https://github.com/Homebrew/homebrew-cask/pull/61994#issuecomment-483862532).

In the future, please use more descriptive commit messages. We have thousands of casks, “reintroduce the deleted cask” isn‘t clear.

**Pull request closed without merging**